AI-and-Analytics/End-to-end-Workloads/LidarObjectDetection-PointPillars/CMakeLists.txt

```diff
@@ -22,11 +22,11 @@ if (InferenceEngine_FOUND)
 endif()
 
 if (NOT InferenceEngine_FOUND)
-  message(STATUS "ERROR: OpenVINO with version 2021.2 or newer required.")
+  message(STATUS "ERROR: OpenVINO™ with version 2021.2 or newer required.")
```

Comment:
Do CMake files support rich text like this?  Do we have other examples of this?  If not, this seems like a risky change, since the CI piepline is unreliable.

Reply:
removed for now